mappings/net/minecraft/entity/ai/memory/MemoryTypes.mapping

```diff
@@ -0,0 +1,2 @@
+CLASS anw net/minecraft/entity/ai/memory/MemoryTypes
```

Comment:
The vanilla name is `MemoryModuleType`. All types are registered here, but this class is also the module class itself. This was also mapped in my PR, so it can be removed.